.github/workflows/scripts/dependabot-pr.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex
+
+git config user.name $GITHUB_ACTOR
+git config user.email $GITHUB_ACTOR@users.noreply.github.com
```

Comment:
Will this handle the situation where the workflow starter is using the "Keep my email addresses private" option?  For example, my `users.noreply.github.com` email is `12352919+TylerHelmuth@users.noreply.github.com`.  Will `GITHUB_ACTOR` contain `TylerHelmuth` or `12352919+TylerHelmuth`?  Or does it not matter?

Reply:
i was mistaken, the `GITHUB_ACTOR` is the github actions bot

Reply:
In the workflow you linked it looks like it is the person who triggered the workflow:

<img width="898" alt="image" src="https://user-images.githubusercontent.com/12352919/172889099-4769e3cb-99e4-4c8a-a390-94137e812267.png">

Reply:
Right, sorry this is confusing. The GITHUB_ACTOR is the user that triggered the action, which is used to create the commit, but the PR is using the GITHUB_TOKEN which is the github action bot's token
![Screen Shot 2022-06-09 at 8 57 03 AM](https://user-images.githubusercontent.com/223565/172891989-f91e9812-e067-4d28-bcbb-6d3afefbeca7.png)
![Screen Shot 2022-06-09 at 8 57 12 AM](https://user-images.githubusercontent.com/223565/172892024-250fbac9-cac1-4109-869f-9d8da4ec32f8.png)

Reply:
Either ways it doesn't look like the email matters to github